src/modules/router/services/router.service.ts

```diff
@@ -58,6 +62,30 @@ export class RouterService {
             pairFilter,
             pairsMetadata,
         );
+        pairsMetadata = await this.filterPairsByTradesCount(
+            pairFilter,
+            pairsMetadata,
+        );
+        pairsMetadata = await this.filterPairsByHasFarms(
+            pairFilter,
+            pairsMetadata,
+        );
+        pairsMetadata = await this.filterPairsByHasDualFarms(
```

Comment:
not in scope of this PR, but maybe we can use the builder design pattern. Then the code could look something like this:
```
pairsMetadata = new PairsMetadataBuilder(pairFilter)
    .filterByState()
    .filterByFeeState()
    .filterByLockedValueUSD()
    .filterBYTradesCount()
...
    .build();
```